packages/antd/src/components/pages/auth/components/register/index.tsx

```diff
@@ -0,0 +1,164 @@
+import React from "react";
+import { RefineRegisterPageProps } from "@pankod/refine-ui-types";
+import {
+    Row,
+    Col,
+    Layout,
+    Card,
+    Typography,
+    Form,
+    Input,
+    Button,
+    LayoutProps,
+    CardProps,
+} from "antd";
+import {
+    useTranslate,
+    useRouterContext,
+    useRegister,
+} from "@pankod/refine-core";
+
+import { layoutStyles, containerStyles, titleStyles } from "../styles";
+
+const { Text, Title } = Typography;
+
+type RegisterProps = RefineRegisterPageProps<LayoutProps, CardProps>;
+
+interface IRegisterForm {
+    username?: string;
+    email?: string;
+    password?: string;
+}
+
+export const RegisterPage: React.FC<RegisterProps> = ({
+    submitButton,
+    loginLink,
+    wrapperProps,
+    contentProps,
+    renderContent,
+}) => {
+    const [form] = Form.useForm<IRegisterForm>();
+    const translate = useTranslate();
+    const { Link } = useRouterContext();
+
+    const { mutate: register, isLoading } = useRegister<IRegisterForm>();
+
+    const CardTitle = (
+        <Title level={3} style={titleStyles}>
+            {translate("pages.register.title", "Sign up your account")}
+        </Title>
+    );
+
+    const CardContent = (
+        <Card
+            title={CardTitle}
+            headStyle={{ borderBottom: 0 }}
+            style={containerStyles}
+            {...(contentProps ?? {})}
+        >
+            <Form<IRegisterForm>
+                layout="vertical"
+                form={form}
+                onFinish={(values) => {
+                    register(values);
+                }}
+                requiredMark={false}
+            >
+                <Form.Item
+                    name="email"
+                    label={translate("pages.register.email", "Email")}
+                    rules={[
+                        { required: true },
+                        {
+                            type: "email",
+                            message: translate(
+                                "pages.register.valiEmail",
```

Comment:
```suggestion
                                "pages.register.validEmail",
```
Let's ask the error text to the core team